lib/src/utils/confirmations/transaction_event.dart

```diff
@@ -43,7 +46,17 @@ class TransactionError with _$TransactionError implements Exception {
         userRejected: (_) => 'user rejected',
         unknownAccount: (_) => 'unknown account',
         rpcError: (_) => 'JSON RPC error',
-        other: (other) => other.reason ?? 'other reason',
+        other: (other) {
```

Comment:
@Chralu , about your last comment: the util class completes the detailed message